### PR TITLE
Librdop v8

### PR DESCRIPTION
**Pull request by bartoszWojciechO**

### CODE REVIEW

File: lib-versions.env [hunk 1]
```diff
@@ -4,6 +4,6 @@
 
 # shellcheck disable=SC2034  # used in the build process later
 LIBTELIO_VERSION=v5.0.0
-LIBDROP_VERSION=v7.0.0
+LIBDROP_VERSION=v8.1.1
```
Comment (mariusSincovici) [inline lib-versions.env:7]: shouldn't it be `8.1.0`?

File: fileshare/libdrop/libdrop.go [hunk 2]
```diff
@@ -240,12 +239,14 @@ func (f *Fileshare) start(
 	isProd bool,
 	storagePath string,
 ) error {
+	var autoretryIntervalMs uint32 = 5000
```
Comment (mariusSincovici) [inline fileshare/libdrop/libdrop.go:242]: [small] auto`R`etryIntervalMs, caps `R`?

File: test/qa/test_fileshare.py [hunk 3]
```diff
@@ -852,7 +852,7 @@ def test_clear():
     peer_address = meshnet.PeerList.from_str(sh.nordvpn.mesh.peer.list()).get_internal_peer().ip
 
     sh.nordvpn.fileshare.send("--background", peer_address, f"{workdir}/{test_files[0]}")
-    time.sleep(1)
+    time.sleep(3)
```
Comment (devzbysiu) [inline test/qa/test_fileshare.py:855]: Is the transfer taking longer now to appear on the pear side?

File: go.mod [hunk 4]
```diff
@@ -8,7 +8,7 @@ toolchain go1.22.2
 // NOTE: If you are chaning the binding versions here, keep in mind that you
 // may also need to update versions in `./lib-versions.env` file.
 require (
-	github.com/NordSecurity/libdrop-go/v7 v7.0.0-rc1
+	github.com/NordSecurity/libdrop-go/v8 v8.0.0-20241017064027-670787595588
```
Comment (devzbysiu) [inline go.mod:11]: It would be nice to have a tag as we had in case of v7

File: fileshare/event_manager.go [hunk 5]
```diff
@@ -259,21 +259,6 @@ func (em *EventManager) handleFileDownloadedEvent(event EventKindFileDownloaded)
 			fileStatusInNotification,
 		)
 	}
-
-	em.finalizeFinishedTransfer(transfer)
-}
-
-func (em *EventManager) finalizeFinishedTransfer(transfer *LiveTransfer) {
```
Comment (devzbysiu) [inline fileshare/event_manager.go:266]: Awesome that it's gone now :tada:

**Review by devzbysiu — APPROVED** [review]

**Pull request merged**